docs/sprints/sprint_2.md

```diff
@@ -0,0 +1,75 @@
+
+## Resumo da Sprint
+
+Número da Sprint: 2 <br>
+Data de Início:  29/08/2020<br>
+Data de Término:  05/09/2020 <br>
+Pontos Planejados:  16<br>
+Pontos Entregues:  16<br>
+
+## Objetivos
+
+- Finalização da primeira versão do restante dos docuementos
+- Início da preparação da arquitetura
+
+## Planejamento
+
+**Faltas** 
+
+Eduardo - Relatou problemas de conexão.
+
+**Tarefas**
+
+Tarefa   | Pontos | Responsáveis
+--------- | ------ |
+TS12 - Criar template de issues e PRs |	1 | Gabriela
+TS11 - Preparar ambiente de desenvolvimento backend |	3 | Geovana
+TS10 - Preparar ambiente de desenvolvimento frontend |	3 | Gabriela
+TS09 - Desenvolver protótipo de baixa fidelidade | 3 | Rodrigo e Pedro
+TS08 - Elaborar Especificação Suplementar | 	3 | Lucas e Jaime
+TS07 - Elaborar Documento de Arquitetura  |	3| Estevão e Eduardo
+**Total de Pontos** | **16**
+
+## Resultados
+
+**Faltas** 
+
+Eduardo - sem justificativas <br>
+Jaime - sem justificativas
+
+**Dívidas**
+
+Nenhuma
+
+
+**Pontos Negativos**
+
+- Desbalanço entre os membros de MDS. Alguns mais comunicativos e participativos e outros menos presentes.
+- Pouca comunicação entre os pareamentos. As duplas não opinaram no trabalho das outras.
+- As orientações de teste da Wiki feitas no planejamento não foram seguidas.
+- Faltas de membros de MDS sem justificativa na reunião de sprint.
+- Compartilhar o andamento das entregas desde o início. 
```

Comment:
Acho que o ponto negativo seria o compartilhamento tardio ou ausente dos artefatos por parte de MDS. Do jeito que está, parece que o ponto negativo é compartilhar as entregas desde o início

Reply:
Verdade, vou corrigir.